chainstate/src/rpc/mod.rs

```diff
@@ -40,99 +40,126 @@ use serialization::hex_encoded::HexEncoded;
 #[rpc::describe]
 #[rpc::rpc(server, client, namespace = "chainstate")]
 trait ChainstateRpc {
-    /// Get the best block ID
+    /// Get the best block ID, which is the tip of the blockchain (i.e., longest chain, or mainchain).
     #[method(name = "best_block_id")]
     async fn best_block_id(&self) -> RpcResult<Id<GenBlock>>;
 
-    /// Get block ID at given height in the mainchain
+    /// Get block ID at a given height in the mainchain.
+    ///
+    /// Returns `None` (null) if the block at the given height does not exist.
     #[method(name = "block_id_at_height")]
     async fn block_id_at_height(&self, height: BlockHeight) -> RpcResult<Option<Id<GenBlock>>>;
 
     /// Returns a hex-encoded serialized block with the given id.
+    ///
+    /// Returns `None` (null) if a block with the given id is not found.
+    /// Note that genesis cannot be retrieved with this function.
     #[method(name = "get_block")]
     async fn get_block(&self, id: Id<Block>) -> RpcResult<Option<HexEncoded<Block>>>;
 
-    /// Returns a json-encoded serialized block with the given id.
+    /// Same as get_block, but returns the block information in json format.
     #[method(name = "get_block_json")]
     async fn get_block_json(&self, id: Id<Block>) -> RpcResult<Option<serde_json::Value>>;
 
-    /// Returns a hex-encoded serialized blocks from the mainchain starting from a given block height.
+    /// Returns hex-encoded serialized blocks from the mainchain starting from a given block height.
+    ///
+    /// The number of returned blocks can be capped using the `max_count` parameter.
     #[method(name = "get_mainchain_blocks")]
     async fn get_mainchain_blocks(
         &self,
         from: BlockHeight,
         max_count: usize,
     ) -> RpcResult<Vec<HexEncoded<Block>>>;
 
-    /// Returns the TxOutput for a specified UtxoOutPoint.
+    /// Returns the TxOutput for a specified UtxoOutPoint. The Utxo must be unspent.
```

Comment:
Maybe to match the others, instead of must be unspent:
Returns `None` (null) if the UtxoOutPoint is not found or is already spent.